src/components/textarea/textarea.style.ts

```diff
@@ -6,13 +6,15 @@ import { StyledLabelContainer } from "../../__internal__/label/label.style";
 import InputIconToggleStyle from "../../__internal__/input-icon-toggle/input-icon-toggle.style";
 import BaseTheme from "../../style/themes/base";
 
-export const MIN_HEIGHT = 64;
+export const DEFAULT_MIN_HEIGHT = 64;
 
 export interface StyledTextAreaProps {
   /** When true, label is placed in line an input */
   labelInline?: boolean;
   /** When true, adjusts padding for icon */
   hasIcon?: boolean;
+  /** Specify the mimimum height */
+  minHeight?: number;
```

Comment:
suggestion: for consistency we should probably keep the type as string

maxWidth is a string and we shouldn't really limit the user to using px when they might want relative heights

Reply:
The `minHeight` is used in a comparison against the `scrollHeight`, which is of type number which represent a pixel value.

In order to allow the `minHeight` to be a string and accept relative height units, we would need a helper function that allows us to dynamically convert the CSS relative units into pixels in a way that is accurate within the current rendering context of the document so that we can continue to make the comparison mentioned above which is necessary for the correct behavior of the component.

An example of how that function could look like is the following:
```
function convertHeightToPixels(heightStr, contextElement = document.body) {
    // Create a temporary element
    const tempElement = document.createElement('div');
 
    // Set the height of the temporary element to the value we want to convert
    tempElement.style.height = heightStr;
 
    // Append the element to the provided context element (default is body)
    contextElement.appendChild(tempElement);
 
    // Get the computed height in pixels
    const pixels = parseFloat(getComputedStyle(tempElement).height);
 
    // Remove the temporary element
    contextElement.removeChild(tempElement);
 
    // Return the height in pixels
    return pixels;
}
 
// Example usage:
const heightInPercent = convertHeightToPixels('10%', <Textarea parent element>);
console.log(heightInPercent);  // Output will be 10% of the height of the parent element in pixels.
```
Another thing we have to consider is that we will have to trigger the above function in case of browser resize events, since the `minHeight` representation in pixels will change as well.

While passing the `minHeight` as a string increases the flexibility of the component, I believe it also ads a layer of complexity to this component that maybe is not required? Please let me know your thoughts on this.

Reply:
The level of complexity required due to the Scroll Height needing to be calculated as a fixed number does seem cumbersome. I'll open a ticket in order to do some further investigation on this later. It's frustrating to not be able to support relative heights out of the gate but it is what it is for now

Reply:
We do have the `resizeObserver` hook in Carbon that could be used for the browser resize stuff, so you wouldn't need to write you own from scratch. The above function is like cutting down a whole tree and setting it all on fire just to light a cigar though. I'm OK with what Will has suggested with regards to opening a separate ticket for an investigation though 👍🏻